python/ray/serve/pipeline/tests/test_deployment_node.py

```diff
@@ -88,49 +84,6 @@ def test_simple_deployment_sync(serve_instance):
     assert ray.get(node.get.execute()) == ray.get(handle.get.remote())
 
 
-def test_no_input_node_as_init_args():
```

Comment:
This test is no longer needed because same verification happens at ClassNode level with tests already.